mlir/lib/Dialect/MQTRef/Translation/ImportQuantumComputation.cpp

```diff
@@ -285,5 +451,14 @@ mlir::OwningOpRef<mlir::ModuleOp> translateQuantumComputationToMLIR(
   // Create terminator
   builder.create<mlir::func::ReturnOp>(loc);
 
+  // Run passes
+  mlir::PassManager passManager(context);
+  passManager.addPass(mlir::createMem2Reg());
+  passManager.addPass(mlir::createRemoveDeadValuesPass());
+  passManager.addPass(mlir::createCanonicalizerPass());
+  if (failed(passManager.run(module))) {
+    emitError(loc) << "Failed to run passes";
+  }
```

Comment:
Just putting this here as a question I am not sure I have an answer for: Should we be running these passes here? or should they be running somewhere else? E.g. only as part of the tests?